### PR TITLE
[8.9] Adds CloudFormation deployment method to CSPM docs

### LINKED ISSUE #3493: [8.9] CSPM cloud formation template deployment update
In 8.9, the CSPM integration will have an update to how it gets deployed. Instead of requiring manual configuration, it will be deployed using a cloud formation template (similar to how vuln management is deployed now). 

This necessitates a reworking on the deployment instructions for CSPM that appear in docs. Open questions include whether the current installation methods (of which there are 5) will remain available as backup/alternative methods.

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3493 by adding the new deployment method to the CSPM docs.

Preview: [Get started with CSPM](https://security-docs_3599.docs-preview.app.elstc.co/guide/en/security/master/cspm-get-started.html)

### CODE REVIEW

**Comment by tinnytintin10** [comment]
@benironside I provided some suggestions in this [g doc](https://docs.google.com/document/d/1vUBqr_qcv8PzH-uNMwvUBfE6lrzH-HGEh43ljntbEzA/edit?usp=sharing).

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 1]
```diff
@@ -38,7 +38,26 @@ To set up CSPM for AWS, add the CSPM integration then enable cloud account acces
...
+
+For most users, the simplest option is to use AWS CloudFormation to automatically provision the necessary resources and permissions in your AWS account. Several manual options are also described below.
+
+[discrete]
+[[cspm-set-up-cloudformation]]
+=== CloudFormation (recommended)
+. Name your integration.
+. Under *Setup Access*, select CloudFormation.
+. In a new browser tab, log in as an admin to the AWS account you want to onboard.
+. Return to your Kibana tab. Click *Save and continue* at the bottom of the page.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:51]: It probably doesn't have to be a _tab_ -- could be a separate window?
```suggestion
. In a new browser tab or window, log in as an admin to the AWS account you want to onboard.
. Return to {kib}. Click *Save and continue* at the bottom of the page.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 2]
```diff
@@ -38,7 +38,26 @@ To set up CSPM for AWS, add the CSPM integration then enable cloud account acces
...
+. Review the information modal, then click *Launch CloudFormation*.
+. A CloudFormation template appears in a new tab. You don't need to modify its configuration.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:53]: Is this a browser tab too, or a tab within the UI?

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 3]
```diff
@@ -125,16 +146,18 @@ If you don't provide values for all configuration fields, the integration will u
...
 [discrete]
 [[cspm-use-iam-arn]]
-=== Option 5 - Use an IAM role Amazon Resource Name (ARN)
-An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions.
-Roles do not have standard long-term credentials such as passwords or access keys.
-Instead, when you assume a role, it provides temporary security credentials for your session.
+==== Option 5 - IAM role Amazon Resource Name (ARN)
+An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions. Roles do not have standard long-term credentials such as passwords or access keys. Instead, when you assume a role, it provides temporary security credentials for your session.
+
+To use an IAM role ARN, select `Assume role` under *Preferred manual method*, enter the ARN, and continue to Finish manual setup.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:156]: Bold for most things in the UI that the user selects:
```suggestion
To use an IAM role ARN, select *Assume role* under *Preferred manual method*, enter the ARN, and continue to Finish manual setup.
```

**Review by joepeeples — COMMENTED** [review]
Looks good! Added a few comments and questions, hope they help!

**Pull request merged**